diff --git a/node_modules/@react-native-menu/menu/ios/MenuViewManager.mm b/node_modules/@react-native-menu/menu/ios/MenuViewManager.mm
index 3d149b3..6816e94 100644
--- a/node_modules/@react-native-menu/menu/ios/MenuViewManager.mm
+++ b/node_modules/@react-native-menu/menu/ios/MenuViewManager.mm
@@ -1,6 +1,6 @@
 #import <React/RCTViewManager.h>
 #import <React/RCTUIManager.h>
-#import "RCTBridge.h"
+#import <React/RCTBridge.h>
 
 #ifdef RCT_NEW_ARCH_ENABLED
 // NEW ARCH
diff --git a/node_modules/@react-native-menu/menu/ios/NewArch/MenuView.mm b/node_modules/@react-native-menu/menu/ios/NewArch/MenuView.mm
index a54e619..b4133d8 100644
--- a/node_modules/@react-native-menu/menu/ios/NewArch/MenuView.mm
+++ b/node_modules/@react-native-menu/menu/ios/NewArch/MenuView.mm
@@ -10,7 +10,7 @@
 #import <react/renderer/components/RNMenuViewSpec/EventEmitters.h>
 #import <react/renderer/components/RNMenuViewSpec/Props.h>
 #import <react/renderer/components/RNMenuViewSpec/RCTComponentViewHelpers.h>
-#import "RCTFabricComponentsPlugins.h"
+#import <React/RCTFabricComponentsPlugins.h>
 
 using namespace facebook::react;
 
